reana_server/rest/workflows.py

```diff
@@ -1374,6 +1413,10 @@ def set_workflow_status(workflow_id_or_name, user):  # noqa
           description: Required. New workflow status.
           required: true
           type: string
+          enum:
+            - start
```

Comment:
Hmm, this may look a bit inconsistent, it would be preferable to use either "start/stop/delete" (verbs) or "started/stopped/deleted" (nouns) everywhere ... but I guess this may not be for now.

(... for the API 1.0 stabilisation we could also clean mixing params/options for execution with all-runs/workspace for deletion...)

Reply:
Yeah, this comes from RWC, so not much we can do as of now: https://github.com/reanahub/reana-workflow-controller/blob/f1764ed7101fa9d30516b8687b7e5ef4ae8781fe/reana_workflow_controller/rest/workflows_status.py#L34-L37